Yona/Yona/DataService/Users.swift

```diff
@@ -9,28 +9,36 @@
 import Foundation
 
 struct Users{
-    var userID: String?
+    var userID: String!
     var firstName: String
     var lastName: String
     var mobileNumber: String
     var nickname: String?
     var editLink: String?
-    
+    var confirmMobileLink: String?
+    var getSelfLink: String?
+
     init(userData: UserData) {
         firstName = userData["firstName"] as! String
         lastName = userData["lastName"] as! String
         mobileNumber = userData["mobileNumber"] as! String
         nickname = userData["nickname"] as? String
```

Comment:
In the next step we can move all this keys (_firstName, lastName_) inside the `Constants.swift` file.